src/main/java/be/seeseemelk/mockbukkit/entity/SlimeMock.java

```diff
@@ -0,0 +1,65 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Slime;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+public class SlimeMock extends MobMock implements Slime
+{
+
+	private int size = 1;
+	private boolean canWander = true;
+
+	/**
+	 * Constructs a new {@link SlimeMock} on the provided {@link ServerMock} with a specified {@link UUID}.
+	 *
+	 * @param server The server to create the entity on.
+	 * @param uuid   The UUID of the entity.
+	 */
+	public SlimeMock(@NotNull ServerMock server, @NotNull UUID uuid)
+	{
+		super(server, uuid);
+	}
+
+	@Override
+	public int getSize()
+	{
+		return this.size;
+	}
+
+	@Override
+	public void setSize(int sz)
+	{
+		if (sz < 1)
+		{
+			throw new IllegalArgumentException("Size cannot be less than 1");
+		}
+		if (sz > 127)
+		{
+			throw new IllegalArgumentException("Size cannot be greater than 127");
+		}
```

Comment:
Technically this doesnt throw in Paper but clamps it to those limits. We should consider keeping to the Paper Implementation here

Reply:
I'd prefer this impl. You can't set it outside that range; if you try to, that should be considered a bug. We should have stricter tolerances than Paper since we're a testing framework, not an actual server.

Reply:
While I agree that we should fail things that don't have an effect in Paper, I think this still should not fail, since it actually performs a certain action. We should still throw a warning, but I think we should emulate Papers behavior here

Reply:
When I think about this, I think I agree on Insprill in this circumstance. I also think it's easier for us developers if we have a clear paradigm for this. I think we need to agree to either fully replicate the behavior of Paper, and throw warnings whenever the input is wrong, or to always be more stringent on user input than paper and throw errors if the input is wrong.

Reply:
I also think it's better to be safe with a stricter tolerance than Papermc.